File: CreditCardValidator/javaScript/creditCardValidatorDriver.js
const prompt = require("prompt-sync")();

const {
    getCreditCardType,
    getSumSecondDigitFromRightToLeft,
    getSumDigitInOddPlacesFromRightToLeft,
    getCreditCardValidity
} = require("./creditCardValidator");

let creditCardDetails = prompt("Hello, Kindly Enter Card details to verify: ")

let creditCardType = getCreditCardType(creditCardDetails)
let cardDigitlength = creditCardDetails.length
let isValid = getCreditCardValidity(creditCardDetails)

console.log(`
Credit Card Type: ${creditCardType}
Credit Card Number: ${creditCardDetails}
Credit Card Digit Length: ${cardDigitlength}
Credit Card Validity Status: ${isValid ? "Valid" : "Invalid"}
`)

File: CreditCardValidator/java/CreditCardValidatorDriver.java
import java.util.Scanner;

public class CreditCardValidatorDriver {

    public static void main(String... labaks) {

        Scanner inputCollector = new Scanner(System.in);

        System.out.println("Hello, Kindly Enter Card details to verify");
        String cardDetails = inputCollector.nextLine();

        String creditCardType = CreditCardValidator.getCreditCardType(cardDetails);
        int cardDigitLength = CreditCardValidator.getCreditCardLength(cardDetails);
        boolean isValid = CreditCardValidator.getCreditCardValidity(cardDetails);

        System.out.println("\nCredit Card Type: " + creditCardType);
        System.out.println("Credit Card Number: " + cardDetails);
        System.out.println("Credit Card Digit Length: " + cardDigitLength);
        System.out.println("Credit Card Validity Status: " + (isValid ? "Valid" : "Invalid"));
    }
}